barretenberg/cpp/src/barretenberg/benchmark/goblin_bench/goblin.bench.cpp

```diff
@@ -11,126 +11,132 @@ using namespace benchmark;
 using namespace bb;
 
 namespace {
-void goblin_full(State& state) noexcept
-{
-    bb::srs::init_crs_factory("../srs_db/ignition");
-    bb::srs::init_grumpkin_crs_factory("../srs_db/grumpkin");
 
+class GoblinBench : public benchmark::Fixture {
```

Comment:
The restructuring of this suite is substantial enough that it may be easier to review in isolation rather than via the diff.